maps/map01.txt

```diff
@@ -1,7 +1,10 @@
-#########
-#.......#####
-#.     .    #
-#.g##  .   ##
-#. ##  .  o#
-#.......####
-#########
+##########
+#o.     ##
+ ######  
+      ####
+#### ###..
+   # #. #
+##    #.##
+   #  #..#
+...#.....#
+##########
```

Comment:
It is intended to be a simple training level that will introduce a concept of map wrapping to our user.